tests/e2e/100-build/300-no-implicit-any.ts

```diff
@@ -0,0 +1,12 @@
+import {updateTsConfig} from '../utils/project';
+import {ng} from '../utils/process';
+import {gitClean} from '../utils/git';
+
+
+export default function() {
+  return updateTsConfig(json => {
+    json['compilerOptions']['noImplicitAny'] = true;
+  })
+  .then(() => ng('build'))
+  .then(() => gitClean());
```

Comment:
What is the purpose of `gitClean` here? Other tests that build aren't cleaning afterwards.